src/bin/cli.test.ts

```diff
@@ -1,20 +1,9 @@
-import { execSync } from 'child_process';
+import { describe, expect, it, test } from 'vitest';
 import { readFileSync, writeFileSync } from 'fs';
 import { tmpNameSync } from 'tmp';
 import { OneSchemaDefinition } from '../types';
 import * as yaml from 'js-yaml';
-
-const executeCLI = (command: string): string => {
```

Comment:
The CLI unit tests used to run a separate node process for every CLI invocation, which was slow and messed up test coverage reporting. I refactored to make the CLI runnable as a function (`createProgram`), and am now running that inside the unit tests. I also got rid of using Jest snapshots in this file's tests because now with `createProgram` we can just assert about errors that are thrown, instead of having to check stdout.

Reply:
These tests used to take 10 seconds to run. Now they take milliseconds.